### DISCUSSION TITLE
BBN Headless Resolution

### DISCUSSION THREAD

**Discussion by aviljoenn** [bareboat-necessities/lysmarine_gen#266] [General]
Hi,
I have installed lysmarine-bbn-bullseye_2022-12-21-raspios-arm64.img.xz on a Pi 4.
I am using VNC to access the Pi 4 with no monitor attached.
I have edited /boot/config.txt and added my own screen parameters at the bottom as follows:

### My Normal HDMI Screen Presets. 
framebuffer_width=1920
framebuffer_height=1080
hdmi_force_hotplug=1
hdmi_group=2
hdmi_mode=82
hdmi_cvt 1920 1080 50 3 0 0 0

This works some of the times for me, but about every second or third time I reboot then the windows on my desktop would open up in the top left hand corner of the screen. Then when I reboot again everything works perfectly to the full size of 1080 resolution.
How can I get it to remain consistent at 1080 resolution?
Thanks.

**Comment by aviljoenn** [comment]
I have also tried changing the VNC resolution using sudo raspi-config, but the same behaviour happens at the various resolutions. The windows appear in the top left hand corner of the screen.

**Comment by mgrouch** [comment]
In Bullseye default driver is kms and same
methods for setting resolution people are used to from Buster are not working. 

See number of solutions here:

https://github.com/raspberrypi/Raspberry-Pi-OS-64bit/issues/225

**Comment by mgrouch** [comment]
You can also use HDMI dummy plug

like this

https://www.adafruit.com/product/4247?gclid=EAIaIQobChMI2_6a3eWw_AIVElNyCh26Pgp2EAQYAyABEgL-_vD_BwE

**Comment by mpolazzo** [comment]
in \boot\config.txt (Terminal => sudo nano config.txt)
	Comment this row to run headless:
	# Enable DRM VC4 V3D driver
	dtoverlay=vc4-kms-v3d,cma-128  <================ comment this row
	max_framebuffers=2
It worked for me.
